entries/s/spaceship.com.au.json

```diff
@@ -4,11 +4,10 @@
 		"additional-domains": [
 			"spaceshipinvest.com.au"
 		],
-		"contact": {
-			"facebook": "SpaceshipAU",
-			"twitter": "SpaceshipAU",
-			"email": "hello@spaceship.com.au"
-		},
+		"tfa": [
+			"sms"
```

Comment:
```suggestion
			"sms",
			"custom-software",
		],
		"custom-software": [
			"WhatsApp"
```

Reply:
Wasn't sure whether or not it counted as a different method to SMS so I left it out.

Reply:
I think it's definitely a different method.